site/docs/getting-started.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Getting started
+---
+
+# Getting Started
+
+## Setup
+
+Add the following dependency to your `build.sbt` file:
+
+``` scala
+libraryDependencies += "ch.epfl.scala" %% "library-example" % "@VERSION@"
+```
+
+Further reading:
+
+ - [Getting Started](getting-started.md)
```

Comment:
Infinite loop? :smile:

Reply:
This one is still wrong, by the way.

Reply:
It is just a page that links to itself :) the link is valid.
We can drop it